apps/web/cypress/plugins/index.ts

```diff
@@ -85,7 +89,7 @@ module.exports = (on, config) => {
       );
 
       let templates;
-      if (!settings?.noEnvironment) {
+      if (!settings?.noTemplates) {
```

Comment:
the `noEnvironment` was not used... 
if `noTemplates` flag is set we won't create the templates in Cypress e2e tests...